ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -56,15 +59,51 @@ const PostBreakingNewsLabel = ({
   ) : null;
 };
 
+const TimeStamp = ({ curated }: { curated: string }) => {
+  const {
+    timezone,
+    locale,
+    altCalendar,
+    service,
+    articleTimestampPrefix,
+    articleTimestampSuffix,
+    script,
+  } = useContext(ServiceContext);
+
+  const isRelative = isTenHoursAgo(new Date(curated).getTime());
+
+  return (
+    <TimeStampContainer
```

Comment:
Is there any way we can avoid using the containers, and use the component instead?

Reply:
Heya, the container has quite a bit of functionality we wanted to keep, such as for setting alternative calendars an such.